mmdeploy/apis/openvino/utils.py

```diff
@@ -19,4 +41,9 @@ def get_input_info_from_cfg(deploy_cfg: mmcv.Config) -> Dict[str, List]:
     # The partition is not supported now. Set the id of model to 0.
     model_inputs = get_model_inputs(deploy_cfg)[0]
     input_info = model_inputs['opt_shapes']
+    onnx_config = deploy_cfg.get('onnx_config', None)
+    if onnx_config:
```

Comment:
It seems that these codes make the priority of `input_names` in `onnx_config` greater than that in `backend_config`. If the `input_names` is set in `backend_config` but it differs from that in `onnx_config`, then the names will be replaced by the names in `onnx_config`.

However, generally, we assume that the `onnx_config` is for ONNX export and `backend_config` is for backend operations. Users should ensure that the names are identical by themselves. Is it necessary to make these changes?

Reply:
Yes, in the current version `input_names` from `onnx_config` has a higher priority.

It seems unclear to me that changing the input name in the config requires making changes in three places:
- `onnx_config.input_names`
- `onnx_config.dynamic_axes`
- `backend_config.model_inputs.opt_shapes`

By the way, it seems to me that there is no need to use a specific input name in `backend_config`. We could just write the default shape value to the config:
```
backend_config = dict(
    model_inputs = dict(
        default_shape = [1, 3, 800, 1344]))
```

Reply:
Sure.  we decide to alias `onnx_config` to `ir_config` and read `input_names` from it (guess we might support more IR in the future).
May I commit in this branch?

Reply:
Yes, please.

Reply:
Hi, It seems that `onnx_config.input_names` and `onnx_config.dynamic_axes` might not share the same names.

For example, if I have a model with two inputs, one needs dynamic shapes to support and one does not need (such as binary-ops between scalar and tensor). Names in `input_names` and `dynamic_axes` would be different since we just need to set the axes of the dynamic input.

We have not begun to support models with multiple inputs yet. I am not sure if it is a good idea to require dynamic axes for all inputs even when the input does not need it. And I am not sure if there would be a similar problem on the inference engine side.

Reply:
Okay, thanks for the clarification. Agree with you about dynamic axes.

At the moment (this may change in the near future) the Model Optimizer from OpenVINO requires shape for all inputs. So, I can add a default input shape to `backend_config.model_inputs` and the ability to manually specify the shapes for the input names. It might look like this:
```
backend_config = dict(
    model_inputs = dict(
        default_shape = [...],
        custom_input_name = [...]
    )
)
```